cmd/scheduler/app/options/options.go

```diff
@@ -60,8 +60,6 @@ type ServerOption struct {
 	PrintVersion         bool
 	EnableMetrics        bool
 	ListenAddress        string
-	EnablePriorityClass  bool
-	EnableCSIStorage     bool
```

Comment:
Let's take code compatibility into consideration.

Reply:
Thanks, the params is kept.